.github/workflows/run-aws-tests.yml

```diff
@@ -0,0 +1,71 @@
+name: Run AWS tests
+
+on:
+  pull_request_target:
+    branches: [main]
+    #opened, reopened and synchronize will cause the workflow to fail on forks due to permissions
+    #once labeled, that will then be overridden by the is-collaborator job
+    types: [opened, labeled, synchronize, reopened]
```

Comment:
Do we need all of those triggers here if we're only using `labeled`?

Reply:
IMO yes, as it provides a good DX. Here's why:

## opened
Needs to be here so it will block the PR when opened by a `fork`. It will run, check permissions aren't good, and fail the workflow. That then requires one of us to come and label. Essentially it's what gives the "auto-block" feature we discussed.

## synchronize/reopened
The idea with these, especially `synchronize` is that a change in the branch will cause the tests to automatically rerun (without needing to relabel). 

Again, if the change was made from a non-collaborator, it will block the PR again and one of us will need to label. If it's a change from a collaborator, the tests should just run

Reply:
okay makes sense!